### DISCUSSION TITLE
Cannot read property '_nextBubbleNoFallback' of undefined

### DISCUSSION THREAD

**Discussion by karthikpks** [vercel/next.js#25436] [Help]
**localhost:3000/example-page** is the default page of my side.

I got **"Cannot read property '_nextBubbleNoFallback' of undefined"**  at **next/next-server/server/next-server.ts** in terminal and  "Internal server error" when entered **localhost:3000/example-page**  in browser.

Kindly advice to solve the error. 
Thanks in advance.

**Comment by sajadZmani** [comment]
@karthikpks i have some issue.  r u solved this ?

**Reply by karthikpks** [reply]
@sajadZmani  
![image](https://user-images.githubusercontent.com/16055839/120145556-47868780-c216-11eb-8837-37a1e92902bb.png)
1) go to next.server.js file
2) find const bubbleNoFallback (you can find it line no 58.)
3) then assign false to bubbleNoFallback 
4) npm run dev
works fine for me.

**Reply by j0hannr** [reply]
Thanks! Solved it in my repository. Is there already an official fix?
Issue at [Ropo Engels-Briefe](https://github.com/studio-arrenberg/engels-briefe)
